python/src/LinearBasisFactory_doc.i.in

```diff
@@ -13,15 +16,11 @@ See also
 --------
 BasisFactory, ConstantBasisFactory, QuadraticBasisFactory
 
-Notes
------
-A factory for linear basis of input dimension *dimension*."
-
-// ---------------------------------------------------------------------
-
-%feature("docstring") OT::LinearBasisFactory::build
-"Build the basis.
-
-Returns
--------
-basis : :class:`~openturns.Basis`."
+Examples
+--------
+>>> import openturns as ot
+>>> basis = LinearBasisFactory(2).build()
+>>> f = ot.AggregatedFunction(basis)
+>>> x = [2, 3]
+>>> print(f(x))
+[1,2,3]"
```

Comment:
Wouldn't it be better to print the basis? Isn't there a pretty print that could be used?